### ISSUE TITLE
WGS Instructions Missing

### ISSUE THREAD

**Issue by DarioS** [McGranahanLab/TcellExTRECT#3]
Can you describe any not-obvious tweaks necessary for it to work on whole genome sequencing data in the user manual?

**Comment by rbentham** [comment]
Hi Dario,

T cell ExTRECT unfortunately is only designed to work on WES samples. 

Support for WGS is being considered and may come in a future release. I do not recommend using the current R package on WGS samples as the code as it exists has been specifically designed, written and tested for use on and **only for WES samples**. There are also likely platform specific biases in WGS samples that will not be accounted for and may not even be obvious without extensive testing. 

I will add a note to the README stating clearly that T cell Ex(ome)TRECT is currently a WES only method.

**Comment by dalalsult** [comment]
Hi Robert,
I have a similar query related to using WGS data on the current package. My project involves identifying molecular aberrations in coding regions that associate with non-response in metastatic HER2-positive breast cancer. I have been using the bed file from the sureselect kit for exome capture in this analysis to look at coding regions only. I was wondering if you have a mixed opinion on deciding to use the same exome capture kit to look at T-cell fraction in my WGS-sequenced samples?
I had carried this out already, but was unable to extract T-cell fraction for two of my metastatic samples with the following error:
Error in UseMethod("distinct") : 
  no applicable method for 'distinct' applied to an object of class "NULL"
 
I have attached the cov_df data frame of one sample for your reference. There appears to be 965,020 rows/positions.
[cov_df_NR033M.csv](https://github.com/McGranahanLab/TcellExTRECT/files/12340445/cov_df_NR033M.csv)
